### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by airMeng**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,6 +10,63 @@ Neural Speed is an innovation library designed to provide the efficient inferenc
 
 > Neural Speed is under active development so APIs are subject to change.
 
+## Quick Start
+There are two methods for utilizing the Neural Speed:
+- [Transformer-based API](#How-to-use-Transformer-based-API)
```
Comment (kevinintel) [inline README.md:15]: link back to ITREX and remove "How to use: Transformer-based API" in NeuralSpeed

Comment (kevinintel) [reply README.md:15]: link to supported models when use_llm_runtime= True

File: README.md [hunk 2]
```diff
@@ -346,7 +366,7 @@ Our Neural Speed supports  INT4 / INT8 / FP8 (E4M3, E5M2) / FP4 (E2M1) / NF4 wei
 | NF4 | BF16 / FP16 / FP32 (FP32) | BF16 / FP32 (FP32) | sym (sym) |
 
 
-### 2. Inference LLM
+### 2. Inference
 
 We provide LLM inference script to run the quantized model. Please reach [us](mailto:itrex.maintainers@intel.com) if you want to run using C++ API directly.
```
Comment (kevinintel) [inline README.md:371]: add enabling new models from https://github.com/intel/intel-extension-for-transformers/blob/main/intel_extension_for_transformers/llm/runtime/graph/developer_document.md

Comment (airMeng) [reply README.md:347]: how about where it already is? https://github.com/intel/neural-speed/blob/a262c61ec691f238f3de45c1dc172bf2ceeadc96/README.md#4-contribution

Comment (kevinintel) [reply README.md]: looks good, but please simple descriptions

File: README.md [hunk 3]
```diff
@@ -10,6 +10,39 @@ Neural Speed is an innovation library designed to provide the efficient inferenc
...
+# convert the model directly use model id in Hugging Face. (recommended)
+python scripts/convert.py --outtype f32 --outfile ne-f32.bin EleutherAI/gpt-j-6b
+```
+
+##### 2. Inference
+
+We provide LLM inference script to run the quantized model. Please reach [us](mailto:itrex.maintainers@intel.com) if you want to run using C++ API directly.
+```bash
+#Linux and WSL
+OMP_NUM_THREADS=<physic_cores> numactl -m 0 -C 0-<physic_cores-1> python scripts/inference.py --model_name llama -m ne-q4_j.bin -c 512 -b 1024 -n 256 -t <physic_cores> --color -p "She opened the door and see"
```
Comment (luoyu-intel) [inline README.md:41]: as it's a python call, can it be done on Windows without numactl?

Comment (luoyu-intel) [reply README.md:41]: BTW, we set omp threads by -t parameter, not OMP_NUM_THREADS

**Review by luoyu-intel — APPROVED** [review]
After the review, it makes me feel that neural_speed is not supported on Windows.

File: README.md [hunk 4]
```diff
@@ -10,6 +10,38 @@ Neural Speed is an innovation library designed to provide the efficient inferenc
...
+Run LLM with one-click python script including conversion, quantization and inference.
+```
+python scripts/run.py model-path --weight_dtype int4 -p "She opened the door and see"
+```
+
+#### Quantize and Inference Step By Step
+Besides the one-click script, Neural Speed also offers the detailed script: 1) convert and quantize, and 2) inference.
+
+##### 1. Convert and Quantize LLM
+Neural Speed assumes the compatible model format as [llama.cpp](https://github.com/ggerganov/llama.cpp) and [ggml](https://github.com/ggerganov/ggml). You can also convert the model by following the below steps:
```
Comment (a32543254) [inline README.md:29]: we only compatible for GGUF format from llama cpp

File: README.md [hunk 5]
```diff
@@ -10,6 +10,38 @@ Neural Speed is an innovation library designed to provide the efficient inferenc
...
+
+```bash
+# convert the model directly use model id in Hugging Face. (recommended)
+python scripts/convert.py --outtype f32 --outfile ne-f32.bin EleutherAI/gpt-j-6b
+```
+
+##### 2. Inference
+
+We provide LLM inference script to run the quantized model. Please reach [us](mailto:itrex.maintainers@intel.com) if you want to run using C++ API directly.
```
Comment (a32543254) [inline README.md:38]: do we have neural speed maintainer  email ？

Comment (kevinintel) [reply README.md:40]: we don't have it now

File: README.md [hunk 6]
```diff
@@ -10,6 +10,44 @@ Neural Speed is an innovation library designed to provide the efficient inferenc
 
 > Neural Speed is under active development so APIs are subject to change.
 
+You can refer [this blog](https://medium.com/@NeuralCompressor/llm-performance-of-intel-extension-for-transformers-f7d061556176) to get the performance information.
+
+## Quick Start
+There are two methods for utilizing the Neural Speed:
+1. We provide [Transformer-based API](https://github.com/intel/intel-extension-for-transformers/blob/main/docs/weightonlyquant.md#llm-runtime-example-code) in [ITREX(intel extension for transformers)](https://github.com/intel/intel-extension-for-transformers), but you need to install Intel Extension for Transformers.
```
Comment (lzwhard) [inline README.md:17]: we can also use neural speed python api:
```python
from transformers import AutoTokenizer, TextStreamer
from neural_speed import Model

prompt = "Once upon a time, a little girl"
tokenizer = AutoTokenizer.from_pretrained(model_name, trust_remote_code=True)
inputs = tokenizer(prompt, return_tensors="pt").input_ids
streamer = TextStreamer(tokenizer)

model = Model()
model.init(model_name, weight_dtype="int4", compute_dtype="int8", group_size=32)
outputs = model.generate(inputs, streamer=streamer, max_new_tokens=30, do_sample=True)

```

Comment (zhenwei-intel) [inline README.md:17]: we can also use neural speed python api:
```python
from transformers import AutoTokenizer, TextStreamer
from neural_speed import Model

prompt = "Once upon a time, a little girl"
tokenizer = AutoTokenizer.from_pretrained(model_name, trust_remote_code=True)
inputs = tokenizer(prompt, return_tensors="pt").input_ids
streamer = TextStreamer(tokenizer)

model = Model()
model.init(model_name, weight_dtype="int4", compute_dtype="int8", group_size=32)
outputs = model.generate(inputs, streamer=streamer, max_new_tokens=30, do_sample=True)

```

**Review by zhenwei-intel — APPROVED** [review]

**Pull request merged**